pkgs/applications/networking/cluster/minikube/default.nix

```diff
@@ -17,26 +17,21 @@ stdenv.mkDerivation rec {
     sha256 = "${checksum}";
   };
 
-  buildInputs = [ ];
+  buildInputs = [ makeWrapper kubernetes ];
```

Comment:
Nit: adding `kubernetes` to the inputs is strictly unnecessary, if all you're doing is write a reference to it into the wrapper script.